include/odp/api/spec/packet_io.h

```diff
@@ -790,6 +790,9 @@ typedef struct odp_pktin_vector_capability_t {
 	/** Packet input vector availability */
 	odp_support_t supported;
 
+	/** odp_pktin_vector_config_t::max_size should be power of two. */
+	odp_bool_t size_is_pow2;
```

Comment:
I wonder if this is needed or whether it would be better to adjust the specification of odp_pktin_vector_config_t::max_size so that the pktin shall deliver received packets _no later than_ when the timeout occurs or when it has accumulated max_size packets. In other words, allow the pktin to produce vectors shorter than max_size even when the timeout does not occur. The application anyway has to be able to handle such shorter vectors for the timeout case.